### DISCUSSION TITLE
Some discussion flag in mail notifications please?

### DISCUSSION THREAD

**Discussion by yajo** [community/community#3021] [Discussions] [answered]
It'd be great if the message came from `discussions@github.com` or were CC-ed there or something like that.

This way, I'd be able to filter them in my inbox and move them to my "forums" folder.

**Comment by becca** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/59. 

If you use web notifications we support filtering for Discussions as mentioned in https://github.com/github/feedback/discussions/59#discussioncomment-131070

**Comment by jhcloos** [comment]
i use the regex:

^Message-ID: <([^/]+/[^/]+)/repo-discussions/

for dicussions, and put them in:

GitHub/Discussions/$1

-JimC
-- 
James Cloos <cloos@jhcloos.com>         OpenPGP: 0x997A9F17ED7DAEA6

**Reply by yajo** [reply]
I guess not all mail providers allow that kind of filters. Not on Gmail, at least, AFAIK.
